libs/initScripts/local.js

```diff
@@ -110,6 +111,22 @@ const run = async () => {
         await _registerCnode(ethAccounts, parseInt(serviceCount))
         break
       }
+      case 'deregister-cnode': {
+        const serviceCount = args[3]
+        if (serviceCount === undefined) throw new Error('deregister-cnode requires a service # as the second arg')
+        await _deregisterCnode(ethAccounts, parseInt(serviceCount))
+        break
+      }
+
+      case 'get-accounts': {
+        await _printAccounts(ethAccounts)
+        break
+      }
+
+      case 'print-accounts': {
+        await _getEthContractAccounts(ethAccounts)
+        break
+      }
```

Comment:
lol, its kinda funny that `get-accounts` calls `_printAccounts` and `print-accounts` calls `_get...Accounts`

Reply:
Removed get-accounts and only have printAccounts now w/ consistent naming